src/catalog/property.cpp

```diff
@@ -53,5 +53,14 @@ std::vector<std::unique_ptr<catalog::Property>> Property::copyProperties(
     return propertiesToReturn;
 }
 
+std::vector<std::unique_ptr<Property>> Property::copy(
```

Comment:
I think we have 
`std::vector<std::unique_ptr<catalog::Property>> Property::copyProperties(
    const std::vector<std::unique_ptr<catalog::Property>>& propertiesToCopy)`
Which does the same thing